### DISCUSSION TITLE
How to change Database Server

### DISCUSSION THREAD

**Discussion by Molyleaf** [requarks/wiki#6521] [Help / Questions] [answered]
I have a Wiki.js instance based on MariaDB, I want to change it smoothly into MySQL, What should I do?

**Comment by P4mes4n** [comment]
Just a heads up: V3 will only support PostgreSQL. So if you want to upgrade to it, maybe reconsider (if you can).
Other than that, what you can or should do, depends on how your instance is hosted.
There may be a better way I'm not aware of. Also I havent done this myself, this is just from the top of my head.

1. I would start by setting up the new Wiki.js instance with the new Database.
2. Then in your old wiki create a Backup of your content via a Local Storage Target.
3. You can then import your Backup in your new instance via a Local Storage Target.
4. Once you have all your content and your other settings in place, point your DNS to the new instance and shut off your old one.

**Comment by NGPixel** [comment, marked as answer]
MariaDB and MySQL are using the same connector and are interchangeable. You can simply export and import back your DB into MySQL. Simply point your wiki config.yml to the new DB and it should work.

However, as @P4mes4n mentioned, you should consider using PostgreSQL if planning on upgrading to v3 in the future. It is however not possible to convert a MariaDB/MySQL db to PostgreSQL. You'll need to follow the steps provided in his reply.
